packages/react-instantsearch-hooks-dom/src/widgets/PoweredBy.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { usePoweredBy } from 'react-instantsearch-hooks';
+
+import { PoweredBy as PoweredByUiComponent } from '../ui/PoweredBy';
+
+import type { ClassNames } from '../types';
+import type { PoweredByProps as PoweredByUiComponentProps } from '../ui/PoweredBy';
+
+export type PoweredByProps = Omit<ClassNames<PoweredByUiComponentProps>, 'url'>;
```

Comment:
Not sure we need the `ClassNames` type anymore since it's already a Partial in the UI component.

Reply:
yes, ClassNames can be removed in #3394